lib/Completion/Tests/Integration/Bridge/TolerantParser/ReferenceFinder/ExpressionNameCompletorTest.php

```diff
@@ -96,6 +96,16 @@ public function provideComplete(): Generator
                 ]
             ]
         ];
+
+        yield 'inside class constant declaration' => [
+            '<?php class Foobar { private const FOO; private const BAR = self::F<>  }', [
+                [
+                    'type'              => Suggestion::TYPE_CONSTANT,
+                    'name'              => 'self::FOO',
```

Comment:
Please check if the expectation is correct.

Reply:
why do you suspect it _not_ to be correct?

Reply:
At the first I started with `name => 'FOO'` - the whole `'self::FOO'` caught my attention.

Reply:
hmm, yeah that doesn't seem quite right, did you test it in the editor?

Reply:
Yes,

https://user-images.githubusercontent.com/11404453/206718275-a8e0bc8d-6d79-4524-80ba-277634802213.mp4

but I found that the new test case checks something that works already on master :grin: so I need to add the proper case before refactoring discussed in the other comment.

Reply:
it is an artificial suggestion:

![image](https://user-images.githubusercontent.com/530801/209409103-837d86e8-a89e-4863-8647-dc0e1d883602.png)